apps/api/package.json

```diff
@@ -1,40 +1,41 @@
 {
-  "name": "@recallnet/api",
+  "name": "api",
   "version": "1.0.0",
-  "description": "Solana Trading Simulator Server for hosting trading competitions",
+  "type": "module",
   "private": true,
+  "description": "Solana Trading Simulator Server for hosting trading competitions",
   "main": "dist/src/index.js",
   "scripts": {
     "lint": "eslint . --max-warnings 0",
     "lint:fix": "eslint . --fix",
     "start": "node dist/src/index.js",
     "clean": "rm -rf .turbo node_modules dist",
-    "dev": "ts-node-dev --transpile-only src/index.ts",
+    "dev": "tsx watch src/index.ts",
```

Comment:
Moved to `tsx` because `ts-node-dev` was throwing weird errors now that we're in esmodule territory. Plus, `ts-node-dev` is an ancient project that hasn't been updated in like 5 or more years.